modules/cluster/addons/kube-proxy.yaml

```diff
@@ -101,7 +101,7 @@ spec:
               fieldRef:
                 apiVersion: v1
                 fieldPath: spec.nodeName
-        image: 602401143452.dkr.ecr.${aws_region}.amazonaws.com/eks/kube-proxy:v1.21.2-eksbuild.2
```

Comment:
In theory we should be able to switch to using the `aws_eks_addon` resource for `kube-proxy` now.

We couldn't before as there was an issue that prevented prometheus from scraping metrics from `kube-proxy` - perhaps we should add a test that this could be done...